docs/setup-cluster/elasticsearch-logging-backend.rst

```diff
@@ -64,5 +65,3 @@ on how the cluster will be used:
      }
    }
 
```

Comment:
Remove blank line. (You can automatically apply the correct formatting by running `make -C docs fmt`, or set up your editor to apply `rstfmt` to reStructuredText files on save.)